pkg/rancher-desktop/main/commandServer/settingsValidator.ts

```diff
@@ -492,6 +492,79 @@ export default class SettingsValidator {
     return false;
   }
 
+  protected checkExtensions(
+    mergedSettings: Settings,
+    currentValue: Record<string, string>,
+    desiredValue: any,
+    errors: string[],
+    fqname: string,
+  ): boolean {
+    if (typeof desiredValue !== 'object' || !desiredValue) {
+      errors.push(`${ fqname }: "${ desiredValue }" is not a valid mapping`);
+
+      return false;
+    }
+
+    // makeRE is a tagged template for making regular expressions with /x (i.e.
+    // ignoring any whitespace within the regular expression itself).
+    function makeRE(strings: {raw: readonly string[]}, ...subsitutions: any[]) {
+      const subsitutionSources = subsitutions.map(s => s instanceof RegExp ? s.source : s);
+      const raw = String.raw(strings, ...subsitutionSources);
```

Comment:
Regarding this line, my IDE says:
```
<html>TS2345: Argument of type '{ raw: readonly string[]; }' is not assignable to parameter of type 'TemplateStringsArray'.

Type '{ raw: readonly string[]; }' is missing the following properties from type 'TemplateStringsArray': length, concat, join, slice, and 18 more.
```

So my IDE is having as much trouble as I am understanding that def'n.